redshift/resource_redshift_user_test.go

```diff
@@ -31,6 +31,9 @@ func TestAccRedshiftUser_Basic(t *testing.T) {
 					resource.TestCheckResourceAttr("redshift_user.with_email", "name", "John-and-Jane.doe@example.com"),
 					testAccCheckRedshiftUserCanLogin("John-and-Jane.doe@example.com", "Foobarbaz1"),
 
+					testAccCheckRedshiftUserExists("hashed_password"),
+					testAccCheckRedshiftUserCanLogin("hashed_password", "ez"),
```

Comment:
```suggestion
					testAccCheckRedshiftUserCanLogin("hashed_password", "Foobarbaz2"),
```

Reply:
Github wont let me propose it inline.. but here's one more test proposal (also I haven't been able to test this), to check the `ALTER USER` machinery:

at line 87 of this file, add this: (from ``)

```
	var configUpdate2 = `
resource "redshift_user" "update_user" {
  name = "update_user2"
  connection_limit = 5
  password = "md508d5d11f1f947091b312fb36b25e621f"
  syslog_access = "UNRESTRICTED"
  create_database = true
}
`
```

and at line 118 insert this:

```
			{
				Config: configUpdate2,
				Check: resource.ComposeTestCheckFunc(
					testAccCheckRedshiftUserExists("update_user2"),
                                        testAccCheckRedshiftUserCanLogin("update_user2", "Foobarbaz6"),
					resource.TestCheckResourceAttr("redshift_user.update_user", "password", "md508d5d11f1f947091b312fb36b25e621f"),
				),
},
```